browser/main/NoteList/index.js

```diff
@@ -659,7 +659,7 @@ class NoteList extends React.Component {
     dataApi
       .createNote(storage.key, {
         type: firstNote.type,
-        folder: folder.key,
+        folder: firstNote.folder,
```

Comment:
I think, after you'ver changed the location of the note, the attachments  will not work anymore because they have been copied to the wrong location (line 667 of this file). Can you have a look at that?

Reply:
Good call out - I will test this.

Reply:
@ehhc it seems to work for me, are you able to break this?

Reply:
@jacobherrington to be honest, it was only a guess that it might break. I haven't tried. Maybe it works fine :)
What might be difficult: if you clone a note to a folder in a different storage (is that even possible??).. But also only a guess.. :) maybe everything is fine :)